opentelemetry-api/java-agent/pom.xml

```
@@ -15,6 +15,8 @@
       <module>TrackCustomException</module>
       <module>TelemetryFilteredBaseOnSpanEvents</module>
       <module>extensions</module>
+        <module>TelemetryFilteredBaseOnRequestDuration</module>
+        <module>extensions/FilterSpanBasedOnDuration</module>
```

(just aligning with above)
```suggestion
      <module>TelemetryFilteredBaseOnRequestDuration</module>
      <module>extensions/FilterSpanBasedOnDuration</module>
```